openc3-cosmos-init/plugins/packages/openc3-cosmos-demo/package.json

```diff
@@ -9,8 +9,8 @@
   },
   "dependencies": {
     "@astrouxds/astro-web-components": "7.26.1",
-    "@openc3/js-common": "6.8.2-beta0",
-    "@openc3/vue-common": "6.8.2-beta0"
+    "@openc3/js-common": "workspace:*",
+    "@openc3/vue-common": "workspace:*"
```

Comment:
Yeah this is what I was hoping for with the `workspace` reference. We can avoid the set version numbers and allow the build process to do it.